packages/block-library/src/query-total/index.php

```diff
@@ -31,7 +31,7 @@ function render_block_core_query_total( $attributes, $content, $block ) {
 	}
 
 	$max_rows       = $query_to_use->found_posts;
```

Comment:
Should we also cast this value just to be on the safe side?

Reply:
I confirmed that `found_posts` is always set as `int`. 

What about `$current_page = max( 1, get_query_var( 'paged', 1 ) );`? In particular, I'm worried about `get_query_var` as it has a mixed return value.